presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaWhereClause.java

```diff
@@ -0,0 +1,120 @@
+package io.prestosql.plugin.kafka;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+import io.prestosql.plugin.kafka.util.TestingKafka;
+import io.prestosql.spi.connector.SchemaTableName;
+import io.prestosql.spi.type.BigintType;
+import io.prestosql.spi.type.Type;
+import io.prestosql.testing.AbstractTestQueryFramework;
+import io.prestosql.testing.QueryRunner;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.UUID;
+
+import static org.apache.kafka.clients.producer.ProducerConfig.ACKS_CONFIG;
+import static org.apache.kafka.clients.producer.ProducerConfig.BOOTSTRAP_SERVERS_CONFIG;
+import static org.apache.kafka.clients.producer.ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG;
+import static org.apache.kafka.clients.producer.ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG;
+
+@Test(singleThreaded = true)
+public class TestKafkaWhereClause
+        extends AbstractTestQueryFramework
+{
+    private TestingKafka testingKafka;
+    private String topicName;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        testingKafka = new TestingKafka();
+        topicName = "test_raw_" + UUID.randomUUID().toString().replaceAll("-", "_");
+
+        Map<SchemaTableName, KafkaTopicDescription> extraTopicDescriptions = ImmutableMap.<SchemaTableName, KafkaTopicDescription>builder()
+                .put(new SchemaTableName("default", topicName),
+                        createDescription(
+                                topicName,
+                                "default",
+                                "default." + topicName,
```

Comment:
Do you need that prefix for the topic name?  We don't seem to be doing it in other places.

Reply:
No, it works without. The other tests seem to just name the topic after the table name, but the tpch tables follow the pattern <Schema Name>.<Table Name> to name the topics. I've been following the tpch pattern. Not sure which is better.